src/tz_linux.rs

```diff
@@ -1,6 +1,17 @@
 use std::fs::{read_link, read_to_string};
 
 pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
+    if let Some(tz) = std::env::var_os("TZ") {
```

Comment:
Given that `env::var_os` is due to be made unsafe in Rust 2024, I'm not comfortable with this. If we do support reading from the TX env var, I'd rather it be gated by a feature and exposed via a new entry point.

At least in Artichoke, I'd not use this as is since my application must run its own hooks on env accesses and e.g. allow disabling all env accesses via a feature for sandboxing.

Reply:
I'd make the same argument. Interacting with environment variables is IMHO out-of-scope of this library.